.github/ISSUE_TEMPLATE/new-docker-db-version.md

```diff
@@ -0,0 +1,16 @@
+---
+name: New Docker-DB Version
+about: Upgrade supported Docker-DB version(s)
+title: Add Docker-DB X.Y.Z
+labels: feature, source:exasol
+assignees: ''
+
+---
+
+# Summary
+A new Docker-DB version which we need to support have been released.
+
+Issue with default bucketfs in 7.1 docker-db#64
```

Comment:
This I don't understand

Reply:
Good catch, was a leftover copy & paste left over.